examples/plot_simulate_mpi_backend.py

```diff
@@ -63,7 +63,11 @@
 # ``openmpi``, which must be installed on the system
 from hnn_core import MPIBackend
 
-with MPIBackend(n_procs=2, mpi_cmd='mpiexec'):
+n_procs = 1  # change this to match the number of CPUs on your system
+if n_procs > 1:
+    with MPIBackend(n_procs=n_procs, mpi_cmd='mpiexec'):
+        dpls = simulate_dipole(net, n_trials=1, postproc=False)
+else:  # use default Joblib backend by default
```

Comment:
you shouldn't have to do this in the example. I think it's handled internally

Reply:
I thought it was, but as you also point out below, the relevant line in `parallel_backends` isn't doing what's expected.

I think that should be fixed so this can be changed back to not have an `if-else` clause.